README.md

```diff
@@ -135,6 +135,7 @@ To run as a desktop app:
 
    ```
    npm install electron
+   npm run install:electron
    node_modules/.bin/electron .
```

Comment:
this feels fiddly and likely for folks to screw up. can't we replace all 3 lines with just `npm run electron`?